test/unit/core/dataframe.test.js

```diff
@@ -1,12 +1,61 @@
 import Dataframe from '../../../src/core/dataframe';
 
-// import dataframe from '../../../src/core/dataframe';
-
 describe('src/core/dataframe', () => {
     describe('.getFeaturesAtPosition', () => {
         describe('when dataframe is point type', () => {
-            it('should return an empty list when there are no points at the given position', () => { });
-            it('should return a list containing the features at the given position', () => { });
+            const dataframe = new Dataframe({
+                center: { x: 0, y: 0 },
+                scale: 1,
+                geom: [
+                    0, 0,
+                    1, 1,
+                ],
+                properties: {
+                    id: [1, 2]
+                },
+                type: 'point',
+                size: 2,
+                active: true,
+                metadata: {
+                    columns: [{
+                        name: 'id',
+                        type: 'float'
+                    }]
+                }
+            });
+            const feature1 = {
+                properties: {
+                    id: 1
+                }
+            };
+            const feature2 = {
+                properties: {
+                    id: 2
+                }
+            };
+            const style = {
+                getWidth: () => ({
+                    eval: () => {
+                        return 0.5;
```

Comment:
Use width 1 to allow the "click inside point but not over stroke" test

Reply:
eh? :confused:

Reply:
With the same width and widthStroke the point is fully covered by the stroke and you can not tests the case when the mouse is inside the point but not over the stroke.

Reply:
The inside stroke case is already being tested with x: 1.0, y: 1.0 + 0.999 / 1024

Reply:
:+1: